### DISCUSSION TITLE
Can't find Kicad / Gerber file

### DISCUSSION THREAD

**Discussion by Sch-LikA** [erichelgeson/BlueSCSI-v1#6] [General]
Hi !

I built a few ArdSCSIno in the past, but the termination added to your board is really interesting. I really like the direction you are leading the project to.

I looked for Kicad or Gerber files, but apart from the note about "Making Gerber files" I can't find it ...

Is it me ?

Thanks !!

**Comment by raybellis** [comment]
I would also love to see the schematic, which is presumably derived from the ztto version.   In particular I'd like to look at doing a version that frees up some of the I2C pins to allow a display to be connected.

**Comment by erichelgeson** [comment]
Hi, Sorry for the late reply - github doesnt seem to notify me of discussions! 

Gerbers an kicad files are here and have always been opensource https://github.com/erichelgeson/bluepill_scsi - forked from a different fork, but I will add them to this repo or add a submodule.

The termination design is derived from RaSCSI v1 boards.

**Reply by Sch-LikA** [reply]
Wonderful, thanks a lot !

**Reply by erichelgeson** [reply]
FYI now moved under the `hw/` folder in this repo.
